flash_examples/image_embedder.py

```diff
@@ -11,31 +11,15 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-import torch
```

Comment:
I would create 2 examples. 1 for ImageEmbedder, and 1 for ImageEmbedder + SSL within integrations under vissl.

Reply:
@tchaton for this PR, I'll keep it the current way, I can add a separate issue and PR to load vissl weights with ImageEmbedder for inference, like I did with the pretrained weights for ImageClassifier